orders/views.py

```diff
@@ -0,0 +1,24 @@
+from django.shortcuts import render
+from .models import OrderItem
+from .forms import OrderCreateForm
+from cart.cart import Cart
+
+
+def order_create(request):
+    cart = Cart(request)
+    if request.method == 'POST':
+        form = OrderCreateForm(request.POST)
```

Comment:
what if the form is  invalid how are you handling that case

Reply:
okk sir, we will add a response for the same